src/type/p5.Font.js

```diff
@@ -669,14 +669,42 @@ class Font {
     // convert lines to paths
     let uPE = this.data?.head?.unitsPerEm || 1000;
     let scale = renderer.states.textSize / uPE;
-    let pathsForLine = lines.map(l => this._lineToGlyphs(l, scale));
+
+    const axs = this._currentAxes(renderer);
+    let pathsForLine = lines.map(l => this._lineToGlyphs(l, { scale, axs }));
 
     // restore the baseline
     renderer.drawingContext.textBaseline = setBaseline;
 
     return pathsForLine;
   }
 
+  _currentAxes(renderer) {
+    let axs;
+    if ((this.data?.fvar?.length ?? 0) > 0) {
+      const fontAxes = this.data.fvar[0];
+      axs = fontAxes.map(([tag, minVal, maxVal, defaultVal, flags, name]) => {
+        if (!renderer) return defaultVal;
+        if (tag === 'wght') {
+          return renderer.states.fontWeight;
+        } else if (tag === 'wdth') {
+          return renderer.states.fontStretch
+        } else if (renderer.canvas.style.fontVariationSettings) {
+          const match = new RegExp(`\\b${tag}\s+(\d+)`)
+            .exec(renderer.canvas.style.fontVariationSettings);
```

Comment:
note to self: should this be the text drawing context's canvas for WebGL? or is there another spot to grab these from?